kyo-actor/shared/src/main/scala/kyo/Actor.scala

```diff
@@ -170,20 +170,20 @@ object Actor:
     /** Receives and processes a single message from the actor's mailbox.
       *
       * This method polls for the next available message and applies the provided processing function. Message processing is done
-      * sequentially, ensuring only one message is handled at a time.
+      * sequentially, ensuring only one message is handled at a time. The result of the processing function is discarded.
       *
       * @param f
       *   The function to process each received message
       * @tparam A
       *   The type of messages being received
       */
-    def receiveAll[A](using Tag[A])[B, S](f: A => B < S)(using Frame): Unit < (Context[A] & S) =
+    def receiveAll[A: Tag](using Frame)[S](f: A => Any < S): Unit < (Context[A] & S) =
```

Comment:
standardizing `Frame` as the implicit parameter used to separate type parameter groups